automated-comments/csharp/general/has_main_method.md

```diff
@@ -0,0 +1,7 @@
+Try removing the `Main` method from code, as the tests don't require them and they make the code slightly harder to read.
+
+However, a better approach is to debug the code while running one or more unit tests. This has the added advantage that one can focus on debugging a specific test case. Here are some links that explain how to debug C# code while running unit tests in various IDE's:
```

Comment:
Is this supposed to be here?

Reply:
Whoops, I missed a bit. Will add the missing text.